### ISSUE TITLE
Abnormal communicative behavior

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7308]
**Preferred term label:**
Abnormal communicative behavior

**Synonyms**

**Definition (free text, please give PubMed ID)**

This would be a grouping term for language anomalies that are considered psychiatric in origin.

**Parent term (use hpo.jax.org/app)**
Behavioral abnormality

**Comment by pnrobinson** [comment]
Added as Abnormal verbal communicative behavior
HP:4000071
